resotool/Makefile

```diff
@@ -13,5 +16,19 @@ migration:
 	$(POETRY_RUN) ./manage.py makemigrations
 	$(POETRY_RUN) ./manage.py migrate
 
+celery-start:
+	$(POETRY_RUN) celery multi start $(CELERY_INSTANCE) -A $(CELERY_MODULE) --loglevel=$(CELERY_LOGLEVEL)
+
+celery-stop:
+	$(POETRY_RUN) celery multi stop $(CELERY_INSTANCE)
+
+celery-run:
+	$(POETRY_RUN) celery -A $(CELERY_MODULE) worker --loglevel=$(CELERY_LOGLEVEL)
+
 server:
-	$(POETRY_RUN) ./manage.py runserver
+	-$(POETRY_RUN) ./manage.py runserver
```

Comment:
Is the dash here intended?

Reply:
That's a relic from some testing I did. The dash should let make ignore errors that occur while executing this line. If it would work as I understood it, we could use it to start a Celery instance as a multi worker, run the server, stop the server witch `Ctrl-c` and stop the Celery worker in just one target. But since it doesn't do what I expected, that doesn't work and I can get rid of the dash when redoing the commit.

Reply:
Yeah, I think I do want to have the return code of this command.